devlib/trace/perf.py

```diff
@@ -113,29 +143,67 @@ def stop(self):
     # pylint: disable=arguments-differ
     def get_trace(self, outdir):
         for label in self.labels:
-            target_file = self._get_target_outfile(label)
-            host_relpath = os.path.basename(target_file)
-            host_file = _f(os.path.join(outdir, host_relpath))
-            self.target.pull(target_file, host_file)
+            # Sometime perf/simpleperf takes a while to write to .data file after kill signal
+            time.sleep(5)
```

Comment:
Remove long fixed-length sleep -- wait on a condition (e.g. probe in a loop with a short sleep between iterations).

Reply:
Let me know what you think about my solution to this.